tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_friendship.c

```diff
@@ -176,7 +182,7 @@ static void friend_wait_for_polls(int polls)
  */
 static void test_friend_est(void)
 {
-	bt_mesh_test_setup();
+	bt_mesh_test_setup(true);
```

Comment:
Probably. However, all tests are actually booting up the stack and initializing it as it would work normally. Therefore, I would say that since mesh stack is practically always used with settings enabled, using this in tests is more close to reality. If there are side effects due to persistence they would get caught.